src/pseudopeople/configuration.py

```diff
@@ -0,0 +1,203 @@
+from pathlib import Path
+from typing import Dict, NamedTuple, Union
+
+import yaml
+from vivarium.config_tree import ConfigTree
+
+from pseudopeople.schema_entities import FORMS
+
+
+class Keys(NamedTuple):
+    """NamedTuple containing all key names used in the configuration file
+    NOTE: 'additional_parameters' is actually a dict with its own key-values defined
+    """
+
+    ROW_NOISE = "row_noise"
+    OMISSION = "omission"
+    DUPLICATION = "duplication"
+    COLUMN_NOISE = "column_noise"
+    ROW_NOISE_LEVEL = "row_noise_level"
+    TOKEN_NOISE_LEVEL = "token_noise_level"
+    ADDITIONAL_PARAMETERS = "additional_parameters"
+
+
+# Define non-baseline default items
+# NOTE: default values are defined in entity_types.RowNoiseType and entity_types.ColumnNoiseType
+DEFAULT_NOISE_VALUES = {
+    # TODO: Uncomment when omission gets implemented
+    # FORMS.CENSUS.name: {
+    #     Keys.ROW_NOISE: {
+    #         Keys.OMISSION: 0.0145,
+    #     },
+    # },
+    # FORMS.ACS.name: {
+    #     Keys.ROW_NOISE: {
+    #         Keys.OMISSION: 0.0145,
+    #     },
+    # },
+    # FORMS.CPS.name: {
+    #     Keys.ROW_NOISE: {
+    #         Keys.OMISSION: 0.2905,
+    #     },
+    # },
+}
+
+
+def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> ConfigTree:
+    """
+    Gets a noising configuration ConfigTree, optionally overridden by a user-provided YAML.
+
+    :param user_configuration: A path to the YAML file or a dictionary defining user overrides for the defaults
+    :return: a ConfigTree object of the noising configuration
+    """
+
+    default_config_layers = [
+        "baseline",
+        "default",
+        "user",
+    ]
+    noising_configuration = ConfigTree(
+        layers=default_config_layers,
+    )
+
+    # Instantiate the configuration file with baseline values
+    baseline_dict = {}
+    for form in FORMS:
+        if not form.is_implemented:
+            continue
+        baseline_dict[form.name] = {}
+        baseline_dict[form.name][Keys.ROW_NOISE] = {}
+        baseline_dict[form.name][Keys.COLUMN_NOISE] = {}
+        for row_noise in form.row_noise_types:
+            if not row_noise.is_implemented:
```

Comment:
After discussion, we decided to remove all not-implemented code. That means removing dummy functions that are currently returning the original col as well as the `is_implemented` attributes in this PR.

(exception being omission b/c this will be implemented very soon and it's the only row noise function)